arches/app/permissions/arches_default_deny.py

```diff
@@ -266,27 +277,23 @@ def get_search_ui_permissions(
 
         user_can_edit = len(self.get_editable_resource_types(user)) > 0
         result["can_edit"] = (
-            user.is_superuser
-            or (
-                groups_edit_exists
-                and len(
-                    set(
-                        search_result["_source"]["permissions"]["groups_edit"]
-                    ).intersection(set(groups))
-                )
-                > 0
-                and user_can_edit
+            groups_edit_exists
```

Comment:
only change here is yanking the is_superuser check since we're already handling for it.